java/src/main/native/CMakeLists.txt

```diff
@@ -39,14 +39,27 @@ option(USE_NVTX "Build with NVTX support" ON)
 option(BUILD_SHARED_LIBS "Build cuDF JNI shared libraries" ON)
 option(BUILD_TESTS "Configure CMake to build tests" ON)
 option(PER_THREAD_DEFAULT_STREAM "Build with per-thread default stream" OFF)
```

Comment:
We should update the option comment here to note that this flag is deprecated.
We should also mark this as advanced https://cmake.org/cmake/help/latest/command/mark_as_advanced.html so the option doesn't show in ccmake or cmake-gui